src/index.html

```diff
@@ -22,6 +22,365 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              data-qa="nav-hover"
+              href="#laptops-computers"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list_item">
+            <a
+              class="nav__list_item-link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__image"></div>
+
+        <div class="card">
+          <h1 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
```

Comment:
This tag used for title of the full page, so will be better if you use the another tag here, for example h2, h3